### DISCUSSION TITLE
Incorrect Temperature 3d Plot Color bar result compared to the temperature device

### DISCUSSION THREAD

**Discussion by SK1GitHub** [firemodels/fds#12901] [General]
I am simulating a model in FDS and while populating the results for temperature 3d plot, the max temperature visible in the results is approx. 400 deg compared to the temperature recorded by the devices at the same location i.e is 1200 deg. I am attaching the FDS file along with the screen shot of the 3d plot and device result. The location of the THCP device is at the surface of the fire block. Kindly suggest remedial measures.
![SCREEN](https://github.com/firemodels/fds/assets/169277743/8875ca17-1425-4eb1-b1ac-e2011549118a)
![untitled](https://github.com/firemodels/fds/assets/169277743/f61db33b-256a-4836-a96a-7cb5303df5a0)
[T1.zip](https://github.com/firemodels/fds/files/15250668/T1.zip)

**Comment by mcgratta** [comment]
Rescale the temperature bar.  Right click, Dialogs, File/Data/Coloring

**Reply by SK1GitHub** [reply]
I have tried this. Doesn't Help.

**Comment by drjfloyd** [comment]
Even after rescaling the colorbar you will not see an exact match, but it should be close. This is for a few reasons:

-Slice files write instanteneous data. Devices write data averaged over the output interval. 
-Slice files by default interpolate cell-centered data like temperature to cell corners to support drawining smooth contours. When corners get reinterpolated to the center by smokeview the value will be close but not idenitical to the original cell centered value. If you want to visualize the actual cell value, add CELL_CENTERED=T to the SLCF input.
-A thermocouple is not the same as gas temperature.  A thermocouple measures the temperature of the thermocouple bead. In radiative environment the bead temperature can be higher or lower than the local gas temperature depending on if the bead sees radiatively hotter or cooler surroundings than the local condition.

**Reply by SK1GitHub** [reply]
I had tried rescaling the color bar, it doesn't get closer to the values registered by the devices. I request for some other probable solution. I used cell centered slices based on your recommendation, but the result and the presentation format that I seek demands the color bar to reflect values atleast around the values of the devices. 
Kindly recommend remedy.
Thank You.

**Comment by drjfloyd** [comment]
The contour image you have in your first post is not from Smokeview and appears to be the Pyrosim native viewer. Do you see the same thing if you use Smokeview?

**Reply by SK1GitHub** [reply]
![image](https://github.com/firemodels/fds/assets/169277743/4f7e395d-1e4d-4027-b02d-e5a31aff18ab)
The image is attached for reference. It shows the same temperature issue here on the color bar.

**Reply by drjfloyd** [reply]
You have loaded the boundary temperature in this image, i.,e. BNDF with QUANTITY='WALL TEMPERATURE'. This is the temperature of the solid walls which is not the same as the gas temperature measued by your DEVC. If you were doing the same in Pyrosim, then this is likely your issue.

**Reply by SK1GitHub** [reply]
Ok, let me try to identify and correct the mistake sir.

**Reply by SK1GitHub** [reply]
I tried loading the temp 3d plot on the smoke view. The only time it shows the color bar is when I select the boundary temp. Every other time it doesn't show the temp bar. Also on Pyrosim, I double checked, I was looking at the 3dplot temperature not the surface temp.
Really stuck and confused here.
I have tried this with different models as well, the problem persists.

**Comment by drjfloyd** [comment]
I ran your input file from your first post with the current release.
I opened the the smv file with smokeview. 
I did Load/Unload->Plot3D-> 600 s
I pressed lower case p a couple of times to cycle to the Plot3D contour to temperature
I typed lower case y to turn on the y-plane
I used the cursor keys to move the plane to the center of the fire.
The attached image is what I see in smokeview with ~1000 C near the top of the burner.
If you follow these steps, do you see the same thing with smokeview?  If so then your issue is with Pyrosim and you need to contact Thunderhead Engineering for assistance as Pyrosim is their product.

![T1_s0000](https://github.com/firemodels/fds/assets/12799217/ad6b2091-08df-4418-89fa-d60dc1fd0497)

**Reply by SK1GitHub** [reply]
Grateful for the effort and the detailed explanation Dr Flyod. I got what you explained above. 
I guess the issue remains with the Pyrosim Software. 
![Smoke View](https://github.com/firemodels/fds/assets/169277743/79fcec2a-3f44-4af2-a080-f9cc92f3cd99)

**Comment by Er9y714** [comment]
I think the defaults bounds in Pyrosim are not max/min but certain percentage. Try choosing the different settings in scale bar settings.
![image](https://github.com/firemodels/fds/assets/26166281/bf36ed6f-0d96-47df-bfea-fbacd60a3adb)
